lxc_ssh.py

```diff
@@ -517,23 +517,6 @@ def __init__(self, play_context, new_stdin, *args, **kwargs):
         self.user = self._play_context.remote_user
         self.control_path = None
         self.control_path_dir = None
-        self.lxc_version = None
-
-        # LXC v1 uses 'lxc-info', 'lxc-attach' and so on
-        # LXC v2 uses just 'lxc'
-        (returncode2, stdout2, stderr2) = self._exec_command("which lxc", None, False)
-        (returncode1, stdout1, stderr1) = self._exec_command(
-            "which lxc-info", None, False
-        )
-        if returncode2 == 0:
-            self.lxc_version = 2
-            display.vvv("LXC v2")
-        elif returncode1 == 0:
-            self.lxc_version = 1
-            display.vvv("LXC v1")
-        else:
-            raise AnsibleConnectionFailure("Cannot identify LXC version")
```

Comment:
This removes a check which is there to check that the version is known, and works.
Future versions might change the behavior.

Reply:
Why not add checks when needed?

Reply:
Anyway, the check cannot go there. It needs to be fired off later, after the _init() and I'm not sure where that is.

Reply:
me neither ;)